sources/packages/web/src/services/http/dto/Institution.dto.ts

```diff
@@ -1,9 +1,9 @@
-import { ClientIdType, DesignationAgreementStatus } from "@/types";
 import {
-  AddressDetailsFormAPIDTO,
   AddressAPIOutDTO,
-  AddressInfoAPIOutDTO,
-} from "@/services/http/dto";
+  ClientIdType,
+  DesignationAgreementStatus,
+} from "@/types";
```

Comment:
Can we move the `AddressAPIOutDTO` and `ContactInformationAPIOutDTO` to the DTO folder?
I belive that anything named DTO should not exists outside the DTO folder.
If needed, we can create a contract we the same schema at the types folder.